socioeconomic-data/data-sources.md

```diff
@@ -29,81 +29,47 @@ New content is announced on the [home page](/) and [news page](/about/news), so
 
-List of standards, schemas, ontologies and/or vocabularies that are relevant to the resources in this category. Consider having links to the "Data description" page.
-When applicable, consider adding the appropriate FAIRsharing entry to the tools and resources spreadsheet so that they can be listed and appropriately tagged at the end of the page
+## Relevant data sources
 
-### Considerations
-
-Using a bullet point style list format as much as possible, describe the considerations for your topic that are specific to infectious diseases within the category (e.g. pathogen characterisation) that you have selected.
-
-Please avoid replicating 'generic' guidelines, i.e. those not specific to infectious diseases, here. Add links to RDMkit in the metadata above, if any are needed. 
-
-### Existing approaches
+<!--- Subsection related to a specific topic related to the data sources of the category that you selected.--->
 
-Mention existing appreaches related to the topic.
+Currently, the COVID-19 Data Portal contains over 537 metadata records from the [Consortium of European Social Science Data Archives' (CESSDA)](https://www.cessda.eu/) [CESSDA Data Catalogue (CDC)](https://datacatalogue.cessda.eu/), which will soon be joined by data from the [European University Institute's (EUI)](https://www.eui.eu/en/home) [COVID-19 SSH Data Portal](https://eui-covid-data.cottagelabs.com/). BY-COVID also aims to add additional data sources over time. 
 
-### Licenses
+The relevant work in the BY-COVID project is performed in the context of activities related to relevant socio-economics data sources for infectious disease outbreaks led by KNAW-DANS and with the participation and expertise of TAU-FSD, EKKE, and EUI. KNAW-DANS, TAU-FSD and EKKE participate in the project as affiliated entities of CESSDA. 
 
-<!--- Example of topic, optional --->
+BY-COVID uses two primary, and multiple secondary data sources for socioeconomic data, which will be outlined in this section. Both primary and secondary data sources provide data that relate to or are relevant for COVID-19 and other infectious disease outbreaks, and therefore the BY-COVID project. The two primary data were determined early on the project, while the list of secondary sources will be progressively identified throughout the course of the project. These data sources are filtered for data relevant to the project, and the resulting metadata is prepared and harmonized via a [harvesting tool](https://docs.google.com/document/d/1a5mbLb3rNVTofD4jTTeIqN3qJnb2QhBI/edit#) before being added to the COVID-19 Data Platform (for more details, see project deliverable [D2.1](https://docs.google.com/document/d/1a5mbLb3rNVTofD4jTTeIqN3qJnb2QhBI/edit#heading=h.7ngyrwsprxyn)).
```

Comment:
```suggestion
BY-COVID uses two primary, and multiple secondary data sources for socioeconomic data, which will be outlined in this section. Both primary and secondary data sources provide data that relate to or are relevant for COVID-19 and other infectious disease outbreaks, and therefore the BY-COVID project. The two primary data were determined early on the project, while the list of secondary sources will be progressively identified throughout the course of the project. These data sources are filtered for data relevant to the project, and the resulting metadata is prepared and harmonized via a [harvesting tool](https://t2-4.by-covid.bsc.es/jspui/) before being added to the COVID-19 Data Platform (for more details, see project deliverable [D2.1](https://zenodo.org/record/7017728).
```
Changed the links to the webpage of the harvester and the zenodo DOI of the deliverable